src/sass/modules/sidebar.scss

```diff
@@ -1,10 +1,18 @@
 .sidebar {
   border-left: 1px solid $black-90;
   max-width: 400px;
+  min-width: 400px;
   overflow-y: auto;
   padding: 16px 20px;
 }
 
+.sidebar--small {
+  @extend .sidebar;
+
+  max-width: 256px;
```

Comment:
Yes, it works but makes me a little nervous that it may effect widths of other sidebars somehow. I'll have to go through each one and verify things weren't changed for them.